### DISCUSSION TITLE
Ya subieron la Versión nueva del Juego Sky.

### DISCUSSION THREAD

**Discussion by RipperxX23** [thatskymod/Sky-CotL-Scripts#137] [Q&A]
Ya la versión nueva de Sky la subieron. Descargue la nueva versión de Canvas 1.5.5 cuando abro ,en la opción de Live no aparece el juego Instalado. En la versión vieja 1.4.4 si aparece, pero cuando abro el juego en Live se cierra la App. ¿Para cuando montan la nueva versión del juego instalado en el Canvas? Gracias.

**Comment by thatskymod** [comment]
¡Gracias por comunicarte con nosotros! Vamos a solucionar el problema.

Asegúrate de que estás ejecutando Sky [0.27.0](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/1356/), Canvas [1.5.6](https://github.com/thatskymod/Sky-CotL-Scripts/blob/754d38203696c8feea8dff40c01f492befc6d85f/Canvas/Canvas.apk) y libLangit [0.4.4](https://github.com/thatskymod/Sky-CotL-Scripts/blob/05fa329f2e50de2d33b6232b5005705f45e5ac91/Canvas/Libs/libLangit044.so).  Esta combinación debería resolver el problema de bloqueo inmediato.

Se pueden encontrar listas de cambios detalladas en el [Foro de discusión de Langit](https://t.me/dukunlangit/8/40577), la [página de lanzamientos](https://github.com/RomanChamelo/Canvas-Open-Source/releases/latest) de Canvas y las [notas del parche](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/1356/) de thatgamecompany.

<details>
  <summary>English version (click to expand)</summary>
  
  Thanks for reaching out! Let's troubleshoot the issue.

Please ensure that you're running Sky [0.27.0](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/1356/), Canvas [1.5.6](https://github.com/thatskymod/Sky-CotL-Scripts/blob/754d38203696c8feea8dff40c01f492befc6d85f/Canvas/Canvas.apk), and libLangit [0.4.4](https://github.com/thatskymod/Sky-CotL-Scripts/blob/05fa329f2e50de2d33b6232b5005705f45e5ac91/Canvas/Libs/libLangit044.so). This combination should resolve the immediate crashing.

Detailed change lists can be found at [Langit Discussion Board](https://t.me/dukunlangit/8/40577), the Canvas [releases page](https://github.com/RomanChamelo/Canvas-Open-Source/releases/latest) and thatgamecompany's [patch notes](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/1356/).
  
</details>
